src/edu/csus/ecs/pc2/ui/EditProblemPane.java

```diff
@@ -425,19 +425,19 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
      * message "Problem Modified - Save Changes?") and responding "Yes" to the message.
      * 
```

Comment:
I would add **javadoc** to the method comment to indicate what this method returns now.  eg.
`@return true if problem was added, false if not`

Reply:
Agree; comments should be updated to reflect new functionality.